src/core/cgroup.c

```diff
@@ -34,6 +35,13 @@
  * out specific attributes from us. */
 #define LOG_LEVEL_CGROUP_WRITE(r) (IN_SET(abs(r), ENOENT, EROFS, EACCES, EPERM) ? LOG_DEBUG : LOG_WARNING)
 
+uint64_t tasks_max_scale(const TasksMax *tasks_max) {
+        if (tasks_max->scale == 0)
```

Comment:
hmm 0% sounds like something one might actually want to specify, i.e. i am pretty sure we should allow everything in the range 0%…100% here, and use UINT64_MAX as the special marker when absolute values are used. i mean, philosophically there's always the question whether 0 or UINT64_MAX should be the special "does not apply" marker, but I'd lean towards the latter here in this case, since 0% sounds like something one might actually want in some context.

Reply:
It is possible to specify 0% as any value. This is just a fraction, so anything with 0 in numerator and non-zero in the denominator is equal to 0%.